src/drivers/intel/cavs/hda-dma.c

```diff
@@ -959,6 +959,9 @@ static int hda_dma_get_attribute(struct dma *dma, uint32_t type,
 	case DMA_ATTR_ALIGNMENT:
 		*value = PLATFORM_HDA_BUFFER_ALIGNMENT;
 		break;
+	case DMA_ATTR_COPY_ALIGNMENT:
+		*value = PLATFORM_HDA_BUFFER_ALIGNMENT;
+		break;
 	default:
```

Comment:
What with dw-dma? It will return error now.

Reply:
Updated